frontend/src/metabase/css/components/modal.css

```diff
@@ -10,6 +10,10 @@
 .Modal.Modal--medium  { width: 65%; }
 .Modal.Modal--wide    { width: 85%; }
 
+.Modal.Modal--tall {
```

Comment:
Ah derp. Yeah fair point. I'm not against using min-height in that case, not sure if we want to do this as part of the modal or maybe just do this on the content inside in this case. I'm starting to get the "we have a lot of modal variants" itches.